src/components/ViewPhotosDialog/index.tsx

```diff
@@ -0,0 +1,132 @@
+import React, { useEffect, useRef, useState } from 'react';
+import DialogBox from '../DialogBox/DialogBox';
+import Button from '../Button/Button';
+import Carousel from 'react-multi-carousel';
+import 'react-multi-carousel/lib/styles.css';
+import { Box, Typography } from '@mui/material';
+import './styles.scss';
+
+export type PhotoItem = {
+  url: string;
+  alt?: string;
+  decoration?: React.ReactNode;
```

Comment:
Component that is rendered below the image, dots, and numbering. Can be a caption, for instance